sync/downstream.py

```diff
@@ -1040,7 +1038,6 @@ def update_pr(git_gecko, git_wpt, sync, action, merge_sha, base_sha, merged_by=N
             # We are storing the wpt base as a reference
             sync.data["wpt-base"] = base_sha
             sync.next_try_push()
-            sync.try_notify()
```

Comment:
Would it not be a good idea to keep this here? try_notify checks if results have already been notified so worse case it is a redundant call that returns immediately. But in case the try push completetion got missed, this would maybe be a useful fallback?

Reply:
No because now the PR is merged before we run try, so we were creating notifications without the try results being available.

Reply:
Yes of course, thanks